apps/autonolas-registry/pages/_document.tsx

```diff
@@ -0,0 +1,46 @@
+import Document, { DocumentContext, Head, Html, Main, NextScript } from 'next/document';
+import { StyleProvider, createCache, extractStyle } from '@ant-design/cssinjs';
+import { ServerStyleSheet } from 'styled-components';
+
+const MyDocument = () => (
+  <Html lang="en">
+    <Head />
+    <body>
+      <Main />
+      <NextScript />
+    </body>
+  </Html>
+);
+
+MyDocument.getInitialProps = async (ctx: DocumentContext) => {
```

Comment:
might be a good candidate for sharing in for example `libs/ui-ssr`? @mohandast52 wdyt